script/os-autoinst-setup-multi-machine

```diff
@@ -17,7 +17,7 @@ ensure_ip_forwarding() {
 
 install_packages() {
     command -v retry > /dev/null || zypper -n in retry
-    retry -- zypper -n in openvswitch os-autoinst-openvswitch firewalld libcap-progs
+    retry -e -s 30 -r 7 -- sh -c "zypper ref && zypper -n in openvswitch os-autoinst-openvswitch firewalld libcap-progs"
```

Comment:
Again, you need to distinguish between installing with custom OBS development repositories or without. AFAIK os-autoinst-setup-multi-machine never states anything about development repositories so it shouldn't care to do those multi-layer retries unless it's an accepted upstream zypper issue that we want to reference and workaround.

Reply:
I'm confused. You literally said `That likely won't prevent the problem. We need to rerun zypper ref` in your previous comment. I guess I'll bring it up on a call

Reply:
yes, but only when *installing with custom OBS development repositories*

Reply:
yes, but only when *installing with custom OBS development repositories*